spec/factories.rb

```diff
@@ -1,4 +1,8 @@
-FactoryGirl.define do
+FactoryGirl.define do  factory :book_author do
```

Comment:
Unnecessary spacing detected.<br>Block body expression is on the same line as the block start.